packages/swapper/src/swappers/zrx/utils/helpers/helpers.ts

```diff
@@ -22,21 +23,25 @@ export const baseUrlFromChainId = (chainId: string): string => {
       return 'https://avalanche.api.0x.org/'
     case KnownChainIds.OptimismMainnet:
       return 'https://optimism.api.0x.org/'
+    case KnownChainIds.BnbSmartChainMainnet:
+      return 'https://bsc.api.0x.org/'
     default:
       throw new SwapError(`baseUrlFromChainId] - Unsupported chainId: ${chainId}`, {
         code: SwapErrorType.UNSUPPORTED_CHAIN,
       })
   }
 }
 
-export const usdcContractFromChainId = (chainId: string): string => {
+export const usdcTokenFromChainId = (chainId: string): string => {
```

Comment:
Prefer `contractAddress` here - `token` / `contract` both lack carrying the notion that this is only the account part of an `AccountId` / `CAIP-19`

Reply:
Also what about
```suggestion
export const usdcTokenFromChainId = (chainId: ChainId): string => {
```

Reply:
I can revert. I changed so it matched the zrx vernacular ie. `buyToken: usdcToken`, but can see your point as well

Reply:
https://github.com/shapeshift/lib/pull/1219/commits/8aadfe5e8efa957692f6bf8b9eafc0b8ece17d12